### PR TITLE
style(ui): typography update

### PR DESCRIPTION
**Pull request by juliendonck**
- [x] remove unused fonts
- [x] allow users to select the system fonts

### CODE REVIEW

**Comment by brandonhaslegs** [comment]
are these supposed to be in snake_case?
image

**Comment by rudolfs** [comment]
> are these supposed to be in snake_case?

yep, in rust variables are snake_case, that's why we have a macro in place which converts them to camelCase for the front-end:

```
#[serde(rename_all = "camelCase")]
```

**Review by rudolfs — DISMISSED** [review]

File: ui/src/settings.ts [hunk 1]
```diff
@@ -31,6 +33,22 @@ interface Option<T> {
   title: string;
 }
 
+export enum Theme {
+  Dark = "dark",
+  Light = "light",
+  H4x0r = "h4x0r",
+}
```
Comment (geigerzaehler) [inline ui/src/settings.ts:40]: This is already imported from `./proxy/settings.ts`. You can remove it here.

File: ui/src/settings.ts [hunk 2]
```diff
@@ -31,6 +33,22 @@ interface Option<T> {
...
+
+export enum UIFont {
+  Inter = "inter",
+  System = "system",
+}
+
+export enum CodeFont {
+  SourceCode = "sourceCode",
+  System = "system",
+}
```
Comment (geigerzaehler) [inline ui/src/settings.ts:50]: These definitions should be moved into `proxy/settings.ts` and you need to update the [`Appearance` schema](https://github.com/radicle-dev/radicle-upstream/blob/b8c4ad4afb781342c3446beebf57ee36e3f42b35/ui/src/proxy/settings.ts#L31-L36) to ensure we properly decode this. (Happy to help if you need support with that).

File: ui/src/settings.ts (hunk 1 above)
Comment (rudolfs) [reply ui/src/settings.ts]: https://github.com/radicle-dev/radicle-upstream/pull/1754/commits/1cfc90ec9a3d1ed94fabbc9431509d5efde31afc

File: ui/src/settings.ts (hunk 2 above)
Comment (rudolfs) [reply ui/src/settings.ts]: https://github.com/radicle-dev/radicle-upstream/pull/1754/commits/1cfc90ec9a3d1ed94fabbc9431509d5efde31afc

File: ui/src/settings.ts (hunk 1 above)
Comment (juliendonck) [reply ui/src/settings.ts]: thx

**Comment by rudolfs** [comment]
Tested on Linux, it works:

https://user-images.githubusercontent.com/158411/114364908-31dfd480-9b7a-11eb-8485-432c4f406679.mov

**Review by rudolfs — DISMISSED** [review]

**Review by geigerzaehler — DISMISSED** [review]

**Review by rudolfs — APPROVED** [review]

**Pull request merged**